src/logger.cr

```diff
@@ -0,0 +1,72 @@
+class Logger
+  property :level, :progname, :formatter
+
+  UNKNOWN = 5
+  FATAL = 4
+  ERROR = 3
+  WARN = 2
+  INFO = 1
+  DEBUG = 0
+
+  SEV_LABEL = %w(DEBUG INFO WARN ERROR FATAL ANY)
```

Comment:
This is just an idea, not sure if it'll be worth it, though.

We can have an enum for this:

```crystal
class Logger
  enum Severity
    DEBUG
    INFO
    WARN
    ERROR
    FATAL
    UKNOWN
  end
end
```

Invoking `to_s` on an enum member gives you its name, so there wouldn't be a need to have a separate array of strings. The formatter would also accept a Severity instead of a String, so it's easier to compare. Then `Logger#level=` would have `Logger::Severity` as a restriction.

We can still have `Logger::DEBUG = Severity::DEBUG` so it's shorter to write.

What do you think?